yarn-project/pxe/src/database/kv_pxe_database.ts

```diff
@@ -611,4 +616,26 @@ export class KVPxeDatabase implements PxeDatabase {
       await Promise.all(senders.map(sender => this.#taggingSecretIndexesForSenders.delete(sender)));
     });
   }
+
+  async store(contract: AztecAddress, key: Fr, values: Fr[]): Promise<void> {
+    if (values.length === 0) {
+      throw new Error('Cannot store an empty array of values in a local contract store');
+    }
```

Comment:
Why not?

Reply:
https://github.com/AztecProtocol/aztec-packages/pull/10867/commits/090cd3233704f0c6b4f832a46edec7ecba5e38a0